pkgs/by-name/yq/yq-go/package.nix

```
@@ -21,13 +22,21 @@ buildGoModule (finalAttrs: {
 
   vendorHash = "sha256-mG9rKla2ZSEbOvSlV6jl7MBoo0dDI//CMcR2hLET4K4=";
 
-  nativeBuildInputs = [ installShellFiles ];
+  nativeBuildInputs = lib.optionalAttrs (stdenv.buildPlatform.canExecute stdenv.hostPlatform) [
+    installShellFiles
+    pandoc
+  ];
 
   postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
     installShellCompletion --cmd yq \
       --bash <($out/bin/yq shell-completion bash) \
       --fish <($out/bin/yq shell-completion fish) \
       --zsh <($out/bin/yq shell-completion zsh)
+
+    patchShebangs ./scripts/generate-man-page*
+    ./scripts/generate-man-page-md.sh
+    ./scripts/generate-man-page.sh
```

How about we also mention the version in the manual page?

https://github.com/mikefarah/yq/blob/76c817009f99310f7719fc006e833b00036e8108/.github/workflows/release.yml#L35

```suggestion
    export MAN_HEADER="yq (https://github.com/mikefarah/yq/) version ${finalAttrs.version}"
    ./scripts/generate-man-page.sh
```